### ISSUE TITLE
streamOperations with filter function

### ISSUE THREAD

**Issue by blueorbitz** [steemit/steem-js#318]
As far as my understanding goes, this stream operation basically return every operation on every user working on the network. Which will be quite heavy on a long run. As most of the cloud service have a bandwidth quota / restriction.

What I am trying to achieve is basically trying to have a real-time response similar to what websocket does.

**Comment by aminraisy** [comment]
exactly, and also functions that give special kind of operations like "transfer" or "custom_json"

**Comment by aminraisy** [comment]
@inertia186 regardless of bandwidth issue through steem-js can we get just "transfer" operations of an account ?
what about through restful APIs documented here https://developers.steem.io ?

**Comment by aminraisy** [comment]
@inertia186 I read your links. thanks alot . but as I understand currently we should get all transaction history and filter them by special op type that we need. no way to directly get just "transfer" type ops even with blockchain API or js libraries. am I right ?

**Comment by ety001** [comment]
This repository has been superseded by a complete 2025 rewrite. The new trunk (`master`, starting at v1.1.0) is a ground-up rewrite of `@steemit/steem-js` and the legacy 0.7.x / 0.8.x code line is archived.

As part of that transition, this issue is being closed because:
- It was filed against the legacy codebase, and the code it references no longer exists in the rewritten trunk.
- The new codebase reimplements the affected areas (RPC routing, transaction serialization, auth/signing, crypto, browser builds) from scratch, so this issue cannot be carried over as-is.

If the problem or request still applies to the rewritten codebase (v1.1.0+ on `master`), please open a new issue against the current trunk with details that reproduce on the new version. Thank you for taking the time to report this originally.
